assembly/compliance-tool/src/main/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/status/ReporterOutput.java

```diff
@@ -0,0 +1,13 @@
+package org.eclipse.sw360.antenna.frontend.compliancetool.sw360.status;
+
+import org.eclipse.sw360.antenna.frontend.compliancetool.sw360.SW360Configuration;
+
+import java.util.Collection;
+
+/**
+ * Central interface for the report output of the status reporter
+ * @param <T>
+ */
+public interface ReporterOutput<T> {
+    void printFile(Collection<T> result, SW360Configuration configuration);
```

Comment:
The use of generics is strange here. A concrete implementation will have to deal with multiple entity types. So the type parameter should be on the method rather than the interface. Alternatively, there could be multiple _printFile()_ methods for different entities.

Or is it expected to have implementations per format and entity class, e.g. ReporterOutputCSVProject, ReporterOutputHtmlRelease, ...?

Reply:
I have not yet implemented the visitor pattern we talked about. 
I felt it better to do these changes in a separate PR when we know exactly what we want.